models/nft/gnosis/nft_gnosis_approvals.sql

```diff
@@ -2,8 +2,8 @@
         alias ='approvals',
         partition_by='block_date',
         materialized='incremental',
-        incremental_strategy='merge',
         file_format = 'delta',
+        incremental_strategy='append',
```

Comment:
Hi @hildobby, do you have any specific reason for the incremental_strategy change to `append`?
As this change may cause the dupes in incremental updates.
https://medium.com/indiciumtech/understanding-dbt-incremental-strategies-part-1-2-22bd97c7eeb5

Reply:
no I'm not sure what is going on here, its a merge issue and the last few fixes commits were me trying to undo those changes, gonna keep trying to fix that